ptero_lsf/implementation/backend.py

```diff
@@ -95,6 +97,13 @@ def create_job(self, command, job_id, options=None, rLimits=None,
 
         return job.as_dict
 
+    def _determine_polling_interval(pollingInterval):
```

Comment:
With this indentation level it is considered a class/instance method and not a global function.  You'll need to use `@staticmethod` decorator and call it with `self._determine_polling_interval(...)` or define it outside of the class scope.